src/pages/index.tsx

```diff
@@ -1,3 +1,20 @@
+import Navigation from 'src/components/Navigation';
```

Comment:
`Navigation` should have its nav folder like `nav`
```suggestion
import Navigation from '@/components/nav/Navigation';
```